packages/gatsby/src/batching/merge-queries.js

```diff
@@ -1,345 +0,0 @@
-const { visit, visitInParallel, Kind } = require(`gatsby/graphql`);
```

Comment:
Are these gatsby file removals supposed to be in here? Why are we removing them?

Reply:
I wiped out all gitignore files. I assume they are here by mistake but @teleaziz maybe let us know if that's a wrong assumption

Reply:
those files looks needed, the gitignore is for batching folder at the root of the package which is being copied on build and publish https://github.com/BuilderIO/builder/blob/f247fa99562f5245f82cbf9dcea1143f4ebf67d3/packages/gatsby/src/constants.js#L5